src/predefinedSimulations.jl

```diff
@@ -139,33 +183,81 @@ function predef_eeg(
 end
 """
 
-    predef_2x2(rng::AbstractRNG; kwargs...)
+    predef_2x2(rng::AbstractRNG; <keyword arguments>)
 
-The most used `kwargs` is: `return_epoched = true` which returns already epoched data. If you want epoched data without overlap, specify `onset = NoOnset()` and `return_epoched = true`
+Simulate data for a 2x2 design i.e. a design with two conditions with two levels each.
 
-#### Design
-- `n_items = 100`,
-- `n_subjects = 1`,
-- `conditions = Dict(:A => ["a_small","a_big"], :B => ["b_tiny","b_large"])`,
-- `event_order_function = shuffle`,
+Note that this function is mainly used for demonstration and internal testing purposes or whenever a quick simulation is needed. \n
 
-#### Component / Signal
-- `signalsize = 100`, # Length of simulated hanning window
-- `basis = hanning(signalsize)`, # The actual "function", `signalsize` is only used here
-- `β = [1, -0.5, .5, +1]`, # The parameters
-- `σs = Dict(:subject => [1, 0.5, 0.5, 0.5],:item => [1])`, # Only in n_subjects >= 2 case, specifies the random effects
-- `contrasts = Dict(:A => EffectsCoding(), :B => EffectsCoding())` # Effect coding by default
-- `formula = n_subjects == 1 ? @formula(0 ~ 1 + A*B) : @formula(dv ~ 1 + A*B + (A*B|subject) + (1|item))`,
+The most used keyword argument is: `return_epoched = true` which returns already epoched data. If you want epoched data without overlap, specify `onset = NoOnset()` and `return_epoched = true`. \n
+Be careful if you modify `n_items` with `n_subjects = 1`, `n_items` has to be a multiple of 4 (or your equivalent conditions factorial, e.g. all combinations length).
```

Comment:
```suggestion
Be careful if you modify `n_items` with `n_subjects = 1`, `n_items` has to be a multiple of 4 (or your equivalent conditions factorial, e.g. all combinations length).

In difference to `predef_EEG`, `predef_2x2` is sample based (no sampling rate to be specified), and also has a 2x2 design, instead of a 2-categorical, 1-continuous design.
```